provider/ansible.rb

```diff
@@ -257,7 +257,7 @@ def generate_resource_tests(data)
 
         name = module_name(data.object)
         path = File.join(target_folder,
-                         "test/integration/targets/#{name}/tasks/main.yml")
+                         "tests/integration/#{name}/tasks/main.yml")
```

Comment:
https://github.com/ansible/ansible_collections_google is showing me a directory structure of `tests/integration/targets/[...]`, is that supposed to match these?

Reply:
you beat me to the punch! I was playing around with the proper structure on the repo and had to backport that change (back).